tests/policy_as_code_test.go

```diff
@@ -80,6 +80,47 @@ func (pc *PolicyAsCodeSuite) TestPolicyAsCode() {
 	pc.deleteCRandObserveInCentral(k8sPolicy, policy.Id)
 }
 
+func (pc *PolicyAsCodeSuite) TestCreateCR() {
+	k8sPolicy := &v1alpha1.SecurityPolicy{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "test-policy-cr",
+			Namespace: "stackrox",
+			OwnerReferences: []metav1.OwnerReference{
+				{
+					APIVersion: "v1alpha1",
+					Kind:       "SecurityPolicy",
+				},
+			},
+		},
```

Comment:
I would remove this whole block and just call `k8sPolicy.SetName("test-policy-cr")`.  You can leave out specifying the namespace because the client is already scoped to that namespace.  I suspect overwriting `ObjectMeta` like this messes with the GVK (group, version kind) used when serializing this object.